src/odb/src/swig/python/dbtypes.i

```diff
@@ -217,7 +217,6 @@
 
 // Handle return by ref.
 %apply int &OUTPUT { int & overhang1, int & overhang2 };
-%apply int &OUTPUT { int & x, int & y };
```

Comment:
Why is this removed?  Are you certain it doesn't affect any db APIs?

Reply:
It was removed, after a very laborious binary search, because it caused SWIG to generate incorrect code when using relatively simple %templates for integer vector recognition in the PdnGen-py.i file. I ran the entire test suite with only this change and without this change and got exactly the same results.

Reply:
Also, note, that this is only for the Python api, not the tcl api.